hamocc/mo_accfields.F90

```diff
@@ -0,0 +1,495 @@
+                                bgct2d,acclvl,acclyr,accsrf,bgczlv,                              &
+                                jatmbromo,jbromo,jbromo_prod,jbromo_uv,jbromofx,jsrfbromo,       &
+                                jcfc11,jcfc11fx,jcfc12,jcfc12fx,jsf6,jsf6fx,                     &
+                                jatmc13,jatmc14,jbigd14c,jcalc13,jco213fxd,jco213fxu,            &
+                                jco214fxd,jco214fxu,jd13c,jd14c,jdic13,jdic14,                   &
+                                jdoc13,jgrazer13,jphyto13,jpoc13,                                &
+                                jlvlnatph,jnatalkali,jnatcalc,jnatco2fx,jnatco3,                 &
+                                jnatdic,jnatomegaa,jnatomegac,jnatpco2,jnatph,                   &
+                                jsrfnatalk,jsrfnatdic,jsrfnatph,                                 &
+                                jbursssc12,jburssso12,jburssssil,jburssster,                     &
+                                jpowaal,jpowaic,jpowaox,jpowaph,jpowaph,jpowasi,jpown2,          &
+                                jpowno3,jsssc12,jssso12,jssssil,jssster,accbur,accsdm,           &
+                                jatmco2,jatmn2,jatmo2
+    use mo_control_bgc,   only: io_stdo_bgc,dtb,use_BROMO,use_AGG,use_WLIN,use_natDIC,           &
+                                use_CFC,use_sedbypass,use_cisonew,use_BOXATM
+    use mo_param1_bgc,    only: ialkali,ian2o,iano3,iatmco2,iatmdms,iatmn2,iatmn2o,iatmo2,       &
+                                icalc,idet,idms,idicsat,idoc,iiron,iopal,                        &
+                                ioxygen,iphosph,iphy,iprefalk,iprefdic,                          &
+                                iprefpo4,iprefo2,isco212,isilica,izoo,                           &
+                                irdin,irdip,irsi,iralk,iriron,irdoc,irdet,inos,iatmbromo,ibromo, &
+                                iatmf11,iatmf12,iatmsf6,icfc11,icfc12,isf6,                      &
+                                iatmc13,iatmc14,icalc13,idet13,idoc13,iphy13,isco213,isco214,    &
+                                izoo13,safediv,                                                  &
+                                iatmnco2,inatalkali,inatcalc,inatsco212,                         &
+                                ipowaal,ipowaic,ipowaox,ipowaph,ipowasi,                         &
+                                ipown2,ipowno3,isssc12,issso12,issssil,issster
+    use mo_sedmnt,        only: powtra,sedlay,burial
+    use mo_vgrid,         only: dp_min
+    use mo_inventory_bgc, only: inventory_bgc
+    use mo_ncwrt_bgc    , only: ncwrt_bgc
+
+    ! Arguments
+    integer , intent(in) :: kpie                  ! 1st dimension of model grid.
+    integer , intent(in) :: kpje                  ! 2nd dimension of model grid.
+    integer , intent(in) :: kpke                  ! size of grid cell (3rd dimension) [m].
```

Comment:
`kpke` is the 3rd (vertical) dimension of the model grid (no units).

Reply:
Thanks so much. I'll fix these this morning. I'm working on the bug fix as well.

Reply:
Done.